e2e/tests/session_steps_test.go

```diff
@@ -241,7 +235,12 @@ func (s *suite) waitForTxResultEvent(targetAction string) {
 	}
 }
 
-func (s *suite) waitForNewBlockEvent(targetEvent string) {
+func (s *suite) waitForNewBlockEvent(
+	targetEvent string,
+	// inspectEventFn passes the matching target event for further validation,
+	// verification, deserialization and enforcement of more granular checks.
+	inspectEventFn func(*abci.Event) bool,
```

Comment:
Wdyt about `eventMatchFn()`, `shouldMatchEvent()`, `isEventMatch()` or something along those lines?

`inspectEventFn()` doesn't make it immediately obvious what it's bool return means.